source/monsters/monsters_a-z/A/index.rst

```diff
@@ -5,5 +5,5 @@ A
    :titlesonly:
    :glob:
 
-   */index
+   **/index
```

Comment:
Ah, i see. It must have been from when i was trying to get it to render sub-indexes. Still never got it working.